tf-psa-crypto/include/psa/crypto.h

```diff
@@ -5168,6 +5168,301 @@ psa_status_t psa_key_agreement_iop_abort(
 
 /**@}*/
 
+/**@}*/
```

Comment:
Double group end, probably a conflict resolution mistake.

Reply:
This has been addressed in the latest [iteration](https://github.com/Mbed-TLS/mbedtls/compare/c699a00aedf08239d93cff1733e80a0dc40d1140..9e088847ae51f8dd76a6fcbd6cd2b21f4a35111b)